pkg/config/app/kuma-cp/config.go

```diff
@@ -74,6 +74,8 @@ type Config struct {
 	Defaults *Defaults `yaml:"defaults"`
 	// Reports configuration
 	Reports *Reports `yaml:"reports"`
+	// Hostname on which Kuma is exposed
+	Hostname string `yaml:"hostname" envconfig:"kuma_hostname"`
```

Comment:
Let's group it under `General` field and move it to the top of `Config`

Reply:
Maybe rename it into one of  `RecommendedHostname`, `AdvertisedHostname`, etc

Reply:
What goes to General? Should Environment also be in General?

Reply:
Let's start by moving only `AdvertisedHostname` into `General`.

And we can open a separate issue for doing the same for `Environment`.